tools/btfgen.sh

```diff
@@ -21,24 +21,26 @@ fi
 # We do not need to add network eBPF programs like dns or snisnoop here because
 # they do not rely on BTF (i.e., they do not include vmlinux.h).
```

Comment:
snisnoop was somehow added back despite the comment.

To detect socket programs that don't use BTF:
```
for f in $(git ls-files |grep 'tracer/.*\.c$') ; do grep -q 'SEC("socket' $f && echo ${f} || true ; done
```